packages/Backend/webpack.config.js

```diff
@@ -47,10 +47,19 @@ module.exports = {
     ]),
     new webpack.DefinePlugin({
       'process.env.NODE_ENV': JSON.stringify(PRODUCTION),
-      'process.env.PG_DATABASE': JSON.stringify(process.env.PG_DATABASE),
-      'process.env.PG_USER': JSON.stringify(process.env.PG_USER),
-      'process.env.PG_PASSWORD': JSON.stringify(process.env.PG_PASSWORD),
-      'process.env.PG_HOST': JSON.stringify(process.env.PG_HOST),
+      'process.env.FB_DATABASE_URL': JSON.stringify(process.env.FB_DATABASE_URL),
```

Comment:
If you do remove that other test file let's remove these TEST_ environmental variables and anything else that pertains to those tests as well.